### ISSUE TITLE
Portfolio Page (Mobile), click image once, one project per line, alt text

### ISSUE THREAD

**Issue by vzhz** [vzhz/vzhz.github.io#9]
Portfolio Page (Mobile):
The Portfolio page is too small and cluttered on mobile.  I'd change it to display one project per line.  

On mobile, the interaction is Click on Portfolio Image -> Get Mouse-Over -> Click Again -> Go to GitHub Page.  I'd change this so you only need to click on the image once.

To do this: Image -> Text Caption -> Text Description/Bullet Points.  Now you no longer need your placeholder image.  You'll probably have to change your css, so that it differentiates between hove and focus/active.  Otherwise, switch your mouse-over to be Javascript, and that should solve the problem.

You can set that text to display: hidden; for desktop, and then have it show up on mobile.  This will also be good for users with screen readers, as they're not going to be able to see your images, and your images don't have any alt text (which I'd recommend putting in descriptive alt text when you have a chance.  I'll even write some for you if you'd like :)

**Comment by vzhz** [comment]
@lunacodes Could to talk about the differences between the JS and hove vs focus/active possibilities?  I know very little JS, but happy to investigate.
I am also have trouble visualizing your recommendation for image/text caption/bullet points.
Will try to catch you soon - just wanted to make a note so I didn't forget/gave you a heads up.

**Comment by lunacodes** [comment]
@vzhz yes, definitely ... and my comment should have been hover, as in a:hover vs. a:focus and a:active

But yeah, let's chat about it together soon :)
